AppController/djinn.rb

```diff
@@ -4695,9 +4713,12 @@ def setup_appengine_application(app, state)
 
       if @app_info_map[app]['nginx'].nil?
         @app_info_map[app]['nginx'] = nginx_app_port
+      end
+      if @app_info_map[app]['nginx_https'].nil?
+        @app_info_map[app]['nginx_https'] = nginx_app_https
```

Comment:
Do you think we should test what happens if this port isn't free? For example, a user might quickly deploy an app before AppScale can restart the old apps on an up.